.github/workflows/publish-components.yaml

```diff
@@ -1,10 +1,13 @@
-on: 
-  push:
+on:
+  pull_request:
```

Comment:
I think the workflow was previously configured to run on `push` because the workflow would run from the context of the forked repo and lose access to secrets used in the workflow when triggered by the `pull_request` event. And it allows pushes of commits directly to `master` to trigger this workflow.

Could we trigger this workflow on `push` and maybe find the associated pull request using a GitHub API request in the workflow in the `pd-component-dev` org?

Reply:
I tried finding a way, but I'll take a deeper look at this.